### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mibes**
For one of our projects we needed the ability to delete documents from an existing index. This extension wraps the "FT.DEL" command with the optional "DD" flag.

### CODE REVIEW

**Comment by mnunberg** [comment]
Can you add a test for this too?

**Comment by mibes** [comment]
Sure. I've just pushed some code that checks if the delete function is working as expected.

File: redisearch/redisearch_test.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/RediSearch/redisearch-go/redisearch"
+	"github.com/mibes/redisearch-go/redisearch"
```
Comment (mnunberg) [inline redisearch/redisearch_test.go]: uh oh!

**Comment by mnunberg** [comment]
Looks good. Please fix the package import and I'll merge. Thanks for the contribution!

File: redisearch/redisearch_test.go (hunk 1 above)
Comment (mibes) [reply redisearch/redisearch_test.go]: Good catch! Should be fixed now.

**Comment by jspc** [comment]
@mnunberg/ @mibes  is there anything left on this PR that I can do/ help with in order to get it merged?

**Pull request merged**